### PR TITLE
use tape for tests and some clean up

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "errno": "./cli.js"
   },
   "devDependencies": {
-    "request": "*"
+    "tape": "^3.5.0"
```
Comment (rvagg) [inline package.json]: `~` please!

**Comment by rvagg** [comment]
good good! note that the stuff in errors.js was generated by a build tool that scraped the libuv headers but I don't think that's even possible any more and we're also slightly out of date with the list in libuv. . oh well!

File: package.json (hunk 1 above)
Comment (ralphtheninja) [inline package.json]: Np!

**Comment by ralphtheninja** [comment]
@rvagg I suspected as such :) If I only had bothered to look more closely! I could follow up on that if want to keep updated with libuv headers.

**Comment by rvagg** [comment]
You could follow it up but I think they are being pulled out of system headers now, I looked in to it at one stage and it was more complicated than it was worth so I gave up. I do think we're a bit out of sync though but I've also not had any complaints!

**Comment by rvagg** [comment]
v0.1.2

**Pull request merged**